bootfiles/bootinst.sh

```diff
@@ -40,5 +40,13 @@ if [ "$DEV" != "$PART" ]; then
    ) | fdisk $DEV >/dev/null 2>&1
 fi
 
+# Now to be able to boot from most of UEFI devices
+EFIDIR="$BOOT/EFI/boot"
```

Comment:
EFI files need to be stored in the root directory of the usb drive. This line would result in EFIDIR=/linux/boot/EFI/boot.

I propose to change line 43 and 44 to (or something similar):
```
mkdir ../../EFI ../../EFI/boot
EFIDIR="$(readlink -f ../../EFI/boot)"
```

Reply:
I think I did this because at some point `BOOT` pointed to usb root directory in Slax bootinst.